assets/stylesheets/base/_form.scss

```
@@ -45,4 +45,14 @@
   &__label {
     @apply text-gray-900 sm:text-sm;
   }
+
+  &--round-input {
```

Minor English preference for:

```suggestion
  &--rounded-input {
```

---

@olivierobert ✅  fixed in [a49e56b](https://github.com/carryall/go-google-scraper-challenge/pull/31/commits/a49e56b28dc2bac8bb22104b13a5c5555416f246)